content/docs/guides/neon-authorize-auth0.md

```diff
@@ -0,0 +1,342 @@
+
+### 5. Install the Neon Serverless Driver
+
+Neon’s Serverless Driver manages the connection between your application and the Neon Postgres database. For Neon Authorize, you must use HTTP. While it is technically possible to access the HTTP API without using our driver, we recommend using the driver for best performance. The driver also supports WebSockets and TCP connections, so make sure you use the HTTP method when working with Neon Authorize.
+
+Install it using the following command:
+
+```bash
+npm install @neondatabase/serverless
+```
+
+To learn more about the driver, see [Neon Serverless Driver](/docs/serverless/serverless-driver).
+
+### 6. Set up environment variables
+
+Here is an example of setting up administrative and authenticated database connections in your `.env` file. Copy the connection strings for both the `neondb_owner` and `authenticated` roles. You can find them from **Connection Details** in the Neon Console, or using the Neon CLI:
+
+```bash
+neonctl connection-string --role-name neondb_owner
+neonctl connection-string --role-name authenticated
+```
+
+Add these to your `.env` file.
+
+```bash shouldWrap
+# Database owner connection string
+DATABASE_URL='<DB_OWNER_CONNECTION_STRING>'
+
+# Neon "authenticated" role connection string
+DATABASE_AUTHENTICATED_URL='<AUTHENTICATED_CONNECTION_STRING>'
+```
+
+The `DATABASE_URL` is intended for admin tasks and can run any query while the `DATABASE_AUTHENTICATED_URL` should be used for connections from authorized users, where you pass the required authorization token. You can see an example in [Run your first authorized query](#2-run-your-first-authorized-query) below.
+
+## Add RLS policies
+
+Now that you’ve integrated Auth0 with Neon Authorize, you can securely pass JWTs to your Neon database. Let's start looking at how to add RLS policies to your schema and how you can execute authenticated queries from your application.
+
+### 1. Add Row-Level Security policies
+
+Below are examples of RLS policies for a **todos** table, designed to restrict access so that users can only create, view, update, or delete their own todos.
+
+<Tabs labels={["Drizzle","SQL"]}>
+
+<TabItem>
+
+```typescript shouldWrap
+import { InferSelectModel, sql } from 'drizzle-orm';
+import { bigint, boolean, pgPolicy, pgTable, text, timestamp } from 'drizzle-orm/pg-core';
+
+export const todos = pgTable(
+  'todos',
+  {
+    id: bigint('id', { mode: 'bigint' }).primaryKey().generatedByDefaultAsIdentity(),
+    userId: text('user_id')
+      .notNull()
+      .default(sql`(auth.user_id())`),
+    task: text('task').notNull(),
+    isComplete: boolean('is_complete').notNull().default(false),
+    insertedAt: timestamp('inserted_at', { withTimezone: true }).defaultNow().notNull(),
+  },
+  (t) => ({
+    p1: pgPolicy('create todos', {
```

Comment:
we should really use crudPolicy here - it's so much simpler

Reply:
these are sticking mostly to what's in the UI - but we dont' have to do that.

Reply:
will update the guides for that next week